### ISSUE TITLE
Keyboard dialog and options for providing instructions for sliders

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#254]
@zepumph, moving from https://github.com/phetsims/gravity-force-lab-basics/issues/171#event-4030016841. I posted this issue there by accident.

Your note in RaP's description design doc referencing differences between a11y content and visual content in the Keyboard Shortcuts dialog, made me think we could have 3 common code options for providing instructions for sliders:

The three common-code options I see are:
1. include all functionality of the slider in the instructions
2. for pedagogical reasons only provide horizontal instructions
3. for pedagogical reason only provide vertical instructions

Hopefully, this is the right place for this issue now, but maybe it should be in a common code repo?

_Originally posted by @terracoda in https://github.com/phetsims/gravity-force-lab-basics/issues/171#issuecomment-731575290_

**Comment by terracoda** [comment]
@zepumph, is this issue clear?
Basically, I am suggesting a path forward for options to slider instructions. 

1. General case for typical sliders, e.g., Resistance in a Wire
![Screen Shot 2020-11-25 at 1 54 24 PM](https://user-images.githubusercontent.com/1902116/100261816-e1970b80-2f25-11eb-9a8d-adabe8c61f0a.png)

2. Special Case Horizontal Emphasis - for pedagogical reasons emphasize horizontal direction in instructions, e.g., GFL regular
![Screen Shot 2020-11-25 at 1 58 36 PM](https://user-images.githubusercontent.com/1902116/100262465-bfea5400-2f26-11eb-9458-ee4e9bc18f88.png)

3. Special Case Vertical Emphasis - for pedagogical reasons emphasize vertical direction in instructions, e.g. GLFB number spinners and RaP individual hands
![gflb-vertical-emphasis](https://user-images.githubusercontent.com/1902116/100262994-81a16480-2f27-11eb-85ae-ec17a7c1af16.png)

![rap-vertical-emphasis](https://user-images.githubusercontent.com/1902116/100263021-89f99f80-2f27-11eb-9199-53f592d3061d.png)

**Comment by zepumph** [comment]
This is clear, and was implemented in commits in common code referenced right below this comment: https://github.com/phetsims/ratio-and-proportion/issues/82#issuecomment-724918145.

The common code component now supports exclusive up/down keys, exclusive left/right, or both. The GFL examples you have above don't use this common code component, but the capabilities to use it exist. I don't think it is worth going back to fix those right now. Anything else for this issue?

**Comment by terracoda** [comment]
@zepumph, sorry, I don't think I was clear. I feel it is important that the common code also have  these three options for the associated the PDOM text.

Currently, in RaP, the PDOM describes all options for sliders rather than only describing the vertical arrows.

I'd like the PDOM text to be as succinct as the visual presentation. 

![Screen Shot 2020-12-10 at 11 58 39 AM](https://user-images.githubusercontent.com/1902116/101792547-64e85d80-3adf-11eb-82dd-85a694b42724.png)

**Comment by zepumph** [comment]
Ok, the description is now being updated in the common code based on what icons we are displaying. Anything else here?

**Comment by terracoda** [comment]
Looks beautiful.
Thanks for the extra effort!

**Comment by terracoda** [comment]
![Screen Shot 2020-12-18 at 5 32 57 PM](https://user-images.githubusercontent.com/1902116/102661296-1e39d980-4157-11eb-91f1-2d05625e6883.png)
